script/build-module.js

```diff
@@ -73,31 +83,49 @@ async function parseFile (file) {
     $('h2').first().text().replace('Class: ', '')
   file.description = $('blockquote').first().text().trim()
 
+  const dirname = path.dirname(file.href)
+
   // fix HREF for relative links
   $('a').each((i, el) => {
     const href = $(el).attr('href')
     const type = hrefType(href)
+
     if (type !== 'relative' && type !== 'rooted') return
-    const dirname = path.dirname(file.href)
-    const newHref = path.resolve(dirname, href.replace(/\.md/, ''))
+
+    let newHref = href
+    if (href.startsWith('../')) {
+      newHref = path.resolve(dirname, href.replace(/\.md/, ''))
+              // need to have this two lines because of
+              // `path.resolve` will return C:\\ (in Windows) if it's root (/)
+              .replace(/C:\\/g, '/')
+              .replace(/\\/g, '/')
+    }
+
     $(el).attr('href', newHref)
   })
 
   // fix SRC for relative images
   $('img').each((i, el) => {
     const baseUrl = 'https://cdn.rawgit.com/electron/electron'
-    const dirname = path.dirname(file.href)
     let src = $(el).attr('src')
     const type = hrefType(src)
+
     if (type !== 'relative' && type !== 'rooted') return
 
     // turn `../images/foo/bar.png` into `/docs/images/foo/bar.png`
-    src = path.resolve(dirname, src)
-
-    const newSrc = file.isApiDoc
+    if (src.startsWith('../')) {
```

Comment:
Since this is used in two places, can you generalize it into function like `resolveRelativePath` or similar?